core/purchasing/inquiry/supplier_allocation_inquiry.php

```diff
@@ -20,20 +20,10 @@
 	$js .= get_js_open_window(900, 500);
 if (user_use_date_picker())
 	$js .= get_js_date_picker();
+$js .= get_js_history(array('supplier_id', 'TransAfterDate', 'TransToDate'));
```

Comment:
Should be:
`$js .= get_js_history(array('supplier_id', 'FromDate', 'ToDate'));`